src/util/voxel_io.cpp

```diff
@@ -69,22 +71,29 @@ from_qb(
     std::array<int32_t, 3>& center, std::array<uint32_t, 3>& size
 )
 {
-    static quill::Logger* logger = logging::get_logger("voxel_io");
+    // static quill::Logger* logger = logging::get_logger("voxel_io");
 
-    LOG_INFO(logger, "Reading voxels from {}", path);
+    LOG_INFO(
+        logging::file_io_logger, "Reading voxels from {}.",
+        path.lexically_normal().string()
+    );
 
     // Read the tiles from the path specified, and save
     std::ifstream file(path, std::ios::in | std::ios::binary);
     if (!file) {
-        LOG_ERROR(logger, "Could not open {}. Are you in the right directory?", path);
+        LOG_ERROR(
+            logging::file_io_logger,
+            "Could not open {}. Are you in the right directory?",
+            path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.